### PR TITLE
update driftignore doc to reflect the use of gitignore and negation syntax

### PR DESCRIPTION
**Pull request by moadibfr**

### CODE REVIEW

**Review by sundowndev — APPROVED** [review]

File: docs/usage/filtering/driftignore.mdx [hunk 1]
```diff
@@ -12,19 +12,14 @@ Create the .driftignore file where you launch driftctl (usually the root of your
...
+- `resource_type.resource_id.path.to.field_name`, resource_id can be wildcard to ignore a drift on given field for a given type, path could also contain wildcards.
 
-:::note
-
-Fields are not case-sensitive.
-
-:::
-
-If your resource id or the path of a field contains dot or backslash you can escape them with backslashes:
+Driftignore also support negation of rules. This way you could ignore everything except certain types.
```
Comment (wbeuil) [inline docs/usage/filtering/driftignore.mdx:17]: "The `.driftignore` file also supports negation of rules."

File: docs/usage/filtering/driftignore.mdx [hunk 2]
```diff
@@ -12,19 +12,14 @@ Create the .driftignore file where you launch driftctl (usually the root of your
...
+For example to ignore everything but iam drift:
```
Comment (wbeuil) [inline docs/usage/filtering/driftignore.mdx:18]: `For example, if you want to ignore everything but IAM drifts:`

File: docs/usage/filtering/driftignore.mdx [hunk 3]
```diff
@@ -35,17 +30,21 @@ aws_s3_bucket.my-buckey
...
+aws_lambda_function.my-lambda-name.last_modified
 ```
 
 ## Precedence over filter rules
 
 The above mechanism to ignore resources can be used in combination with filter rules. Bear in mind that if the same resource is included by a filter rule and excluded inside the .driftignore file, driftctl will just ignore this resource.
 
+## Precedence over resource scan
+
+Driftignore rules are applied after the resources are scanned, so you cannot prevent a resource to be fetched using it.
```
Comment (wbeuil) [inline docs/usage/filtering/driftignore.mdx:46]: `.driftignore rules are applied after the resources are scanned. Thus, you cannot prevent a resource to be fetched using it.`

File: docs/usage/filtering/driftignore.mdx [hunk 4]
```diff
@@ -35,17 +30,21 @@ aws_s3_bucket.my-buckey
...
 # Will ignore lastModified for my-lambda-name lambda function
-aws_lambda_function.my-lambda-name.LastModified
+aws_lambda_function.my-lambda-name.last_modified
 ```
 
 ## Precedence over filter rules
 
 The above mechanism to ignore resources can be used in combination with filter rules. Bear in mind that if the same resource is included by a filter rule and excluded inside the .driftignore file, driftctl will just ignore this resource.
 
+## Precedence over resource scan
```
Comment (wbeuil) [inline docs/usage/filtering/driftignore.mdx:44]: `Precedence over resourceS scan`

**Review by wbeuil — CHANGES_REQUESTED** [review]

File: docs/usage/filtering/driftignore.mdx [hunk 5]
```diff
@@ -12,19 +12,15 @@ Create the .driftignore file where you launch driftctl (usually the root of your
 Each line must be of kind
 
 - `resource_type.resource_id`, resource_id could be a wildcard to exclude all resources of a given type.
-- `resource_type.resource_id.path.to.FieldName`, resource_id can be wildcard to ignore a drift on given field for a given type, path could also contain wildcards.
+- `resource_type.resource_id.path.to.field_name`, resource_id can be wildcard to ignore a drift on given field for a given type, path could also contain wildcards.
 
-:::note
+The .driftignore file also support negation of rules. This way you could ignore everything except certain types.
```
Comment (wbeuil) [inline docs/usage/filtering/driftignore.mdx:17]: `supportS`

**Review by wbeuil — APPROVED** [review]

**Pull request merged**